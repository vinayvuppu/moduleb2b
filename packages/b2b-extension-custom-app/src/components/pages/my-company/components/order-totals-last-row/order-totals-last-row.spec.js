import React from 'react';
import { shallow } from 'enzyme';
import OrderTotalsLastRow from './order-totals-last-row';

const createTestProps = () => ({ total: 20 });

describe('rendering', () => {
  const props = createTestProps();
  const wrapper = shallow(<OrderTotalsLastRow {...props} />);

  it('outputs correct tree', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
