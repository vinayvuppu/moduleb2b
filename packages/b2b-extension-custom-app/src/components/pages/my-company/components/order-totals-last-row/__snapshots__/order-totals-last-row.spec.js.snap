// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering outputs correct tree 1`] = `
<div
  className="container"
  data-testid="order-totals-last-row"
>
  <div
    className="total"
  >
    <span
      className="total-text"
    >
      <FormattedMessage
        defaultMessage="Total"
        description="Total of table label"
        id="OrderTotalsLastRow.totalLabel"
        values={Object {}}
      />
    </span>
    <span
      className="total-amount"
    >
      20
    </span>
  </div>
</div>
`;
